README.md

```diff
@@ -112,6 +112,8 @@ Set up (the latest version of) [HAProxy](http://www.haproxy.org/) in Ubuntu syst
 * `haproxy_listen.{n}.capture.type`: [required]: What to capture (`cookie`, `request header`, `response header`)
 * `haproxy_listen.{n}.capture.name`: [required]: Name of the header or cookie to capture
 * `haproxy_listen.{n}.capture.length`: [required]: Maximum number of characters to capture and report in the logs
+* `haproxy_listen.{n}.filter`: [optional]: Dictionary of content filters to apply to this section
```

Comment:
I think we better not make filter a Dictionary. This makes it impossible the implement the [following example](https://cbonte.github.io/haproxy-dconv/1.7/configuration.html#4-filter):

```yml
listen
  bind *:80

  filter trace name BEFORE-HTTP-COMP
  filter compression
  filter trace name AFTER-HTTP-COMP

  compression algo gzip
  compression offload
```

Reply:
Better a `list` like `haproxy_listen.{n}.http_request` with a `name` and `param` property

Reply:
Ooo, yes, I agree. I'll update to that in the next couple of days

Reply:
Can you have a look, I would like to add this functionality. Thanks in advance.

Reply:
I'd completely forgotten about this, sorry... remind me again in a week if I haven't done anything